src/webgpu/api/validation/image_copy/layout_related.spec.ts

```diff
@@ -250,68 +254,86 @@ g.test('offset_alignment')
   });
 
 g.test('bound_on_bytes_per_row')
+  .desc(`For all formats, verify image copy validations w.r.t bytesPerRow.`)
   .params(u =>
     u
       .combine('method', kImageCopyTypes)
       .combine('format', kSizedTextureFormats)
       .filter(formatCopyableWithMethod)
       .beginSubcases()
-      .combineWithParams([
-        { blocksPerRow: 2, additionalPaddingPerRow: 0, copyWidthInBlocks: 2 }, // success
-        { blocksPerRow: 2, additionalPaddingPerRow: 5, copyWidthInBlocks: 3 }, // success if bytesPerBlock <= 5
-        { blocksPerRow: 1, additionalPaddingPerRow: 0, copyWidthInBlocks: 2 }, // failure, bytesPerRow > 0
-        { blocksPerRow: 0, additionalPaddingPerRow: 0, copyWidthInBlocks: 1 }, // failure, bytesPerRow = 0
-      ])
-      .combineWithParams([
-        { copyHeightInBlocks: 0, copyDepth: 1 }, // we don't have to check the bound
-        { copyHeightInBlocks: 1, copyDepth: 0 }, // we don't have to check the bound
-        { copyHeightInBlocks: 2, copyDepth: 1 }, // we have to check the bound
-        { copyHeightInBlocks: 0, copyDepth: 2 }, // we have to check the bound
-      ])
+      .expand('params', p => {
```

Comment:
You can use `.expandWithParams(p => {` and then each of the members below (`bytesPerRow`, `blocksPerRow`, `copyWidthInBlocks`, `success`) will arrive as a separate parameter.

This makes the query strings a bit nicer, but more importantly it lets you change `success` to `_success` which makes it a "private" parameter which doesn't show up in the query string.

Reply:
Done.